easybuild/easyconfigs/o/ORCA/ORCA-3_0_0-linux_x86-64-OpenMPI-1.6.5.eb

```diff
@@ -1,20 +1,23 @@
 easyblock = "PackedBinary"
 
 name = "ORCA"
-version = '3_0_0-linux_x86-64_openmpi_165'
+version = '3_0_0-linux_x86-64'
```

Comment:
We only have sources for ORCA v3.0.0 named `orca_3_0_0_linux_x86-64_openmpi_165.tbz `, so it seems like in *this* case the `_openmpi_165` should be part of the version after all...